deployability/modules/testing/tests/test_central_components/test_uninstall.py

```diff
@@ -0,0 +1,84 @@
+from ..helpers.constants import WAZUH_ROOT
+from ..helpers.generic import HostInformation, GeneralComponentActions
+from ..helpers.manager import WazuhManager
+from ..helpers.dashboard import WazuhDashboard
+from ..helpers.indexer import WazuhIndexer
+from ..helpers.central import WazuhCentralComponents
+
+
+@pytest.fixture(scope="module", autouse=True)
+def wazuh_params(request):
+    wazuh_version = request.config.getoption('--wazuh_version')
+    wazuh_revision = request.config.getoption('--wazuh_revision')
+    dependencies = request.config.getoption('--dependencies')
+    targets = request.config.getoption('--targets')
+
+    return {
+        'wazuh_version': wazuh_version,
+        'wazuh_revision': wazuh_revision,
+        'dependencies': dependencies,
+        'targets': targets
+    }
+
+
+@pytest.fixture(scope="module", autouse=True)
+def setup_test_environment(wazuh_params):
+    targets = wazuh_params['targets']
+    # Clean the string and split it into key-value pairs
+    targets = targets.replace(' ', '')
+    targets = targets.replace('  ', '')
+    pairs = [pair.strip() for pair in targets.strip('{}').split(',')]
+    targets_dict = dict(pair.split(':') for pair in pairs)
+
+    wazuh_params['master'] = targets_dict.get('wazuh-1')
+    wazuh_params['workers'] = [value for key, value in targets_dict.items() if key.startswith('wazuh-') and key != 'wazuh-1']
+    wazuh_params['indexers'] = [value for key, value in targets_dict.items() if key.startswith('node-')]
+    wazuh_params['dashboard'] = targets_dict.get('dashboard', wazuh_params['master'])
+
+    # If there are no indexers, we choose wazuh-1 by default
+    if not wazuh_params['indexers']:
+        wazuh_params['indexers'].append(wazuh_params['master'])
+
+    wazuh_params['managers'] = {key: value for key, value in targets_dict.items() if key.startswith('wazuh-')}
+
+
+def test_uninstall(wazuh_params):
+    assert 'active' in GeneralComponentActions.get_component_status(wazuh_params['master'], 'wazuh-manager'), logger.error(f'The manager in {HostInformation.get_os_name_and_version_from_inventory(wazuh_params["master"])} is not active')
+    assert 'active' in GeneralComponentActions.get_component_status(wazuh_params['dashboard'], 'wazuh-dashboard'), logger.error(f'The dashboard in {HostInformation.get_os_name_and_version_from_inventory(wazuh_params["dashboard"])} is not active')
+    for indexer_params in wazuh_params['indexers']:
+        assert 'active' in GeneralComponentActions.get_component_status(indexer_params, 'wazuh-indexer'), logger.error(f'The indexer in {HostInformation.get_os_name_and_version_from_inventory(indexer_params)} is not active')
+    assert 'active' in GeneralComponentActions.get_component_status(wazuh_params['master'], 'filebeat'), logger.error(f'The filebeat in {HostInformation.get_os_name_and_version_from_inventory(wazuh_params["master"])} is not active')
+
+    #WazuhCentralComponents.perform_uninstall_and_scan_for_aio(wazuh_params['master'])
```

Comment:
Remove comment

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/5324/commits/96c47257f1bca33c7f144285a4c7b63a8c10fbce